src/app/api/activity-logs/services/activity-log.service.ts

```diff
@@ -69,6 +69,7 @@ export class ActivityLogService extends BaseService {
     const allReplies = await commentService.getReplies(commentIds)
 
     const logResponseData = parsedActivityLogs.map((activityLog) => {
+      const initiator = copilotUsers.find((iu) => iu.id === activityLog.userId)
```

Comment:
Not a big issue but this would make our code slightly more concise

```suggestion
      const initiator = copilotUsers.find((iu) => iu.id === activityLog.userId) || null
```